scripts/install.sh

```diff
@@ -187,14 +187,29 @@ install_dependencies() {
 main() {
     mkdir -p "${PREFIX}/bin"
 
+   # Parse command line arguments
+    local key_type="light"
+    while [[ $# -gt 0 ]]; do
+        case $1 in
+            --full-keys)
+                key_type="full"
+                shift
+                ;;
+            *)
+                echo "Unknown option: $1"
+                exit 1
+                ;;
+        esac
+    done
```

Comment:
for all other ci tests than prover, do we currently download all provingkeys?

Reply:
No, by default we download only subset of proving keys just for tests, I wrote a note about it here https://github.com/Lightprotocol/light-protocol/pull/1295/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R86-R91